src/exts/ajo.py

```diff
@@ -176,13 +180,43 @@ async def __get_leaderboard(self) -> Embed:
         return embed
 
     # LEADERBOARD
-    @command(name="leaderboard", description="Get the ajo leaderboard.")
-    async def leaderboard_command(self, ctx: Context[Bot]) -> None:
-        await ctx.reply(embed=await self.__get_leaderboard())
-
     @slash_command(name="leaderboard", description="Get the ajo leaderboard.")
     async def leaderboard(self, itr: CommandInteraction) -> None:
-        await itr.send(embed=await self.__get_leaderboard())
+        view = LeaderboardCustomView(itr.author)
+        previous = Button(style=ButtonStyle.primary, label="Previous", emoji="⏪", custom_id="p")
+        next =  Button(style=ButtonStyle.primary, label="Next", emoji="⏩", custom_id="n")
+        if view.page == 1:
+            previous.disabled = True
+        view.add_item(previous)
+        view.add_item(next)
+        async def button_callback(button_inter: MessageInteraction):
+            # Increase or decrease page based on the button id
+            if button_inter.component.custom_id == "n":
+                view.page += 1
+            else:
+                view.page -= 1
+
+            # Disable the "previous" button only if we're on the first page
+            is_first_page = view.page == 1
+            previous.disabled = is_first_page
+
+            # Modify the original message with the new page
+            await itr.edit_original_response(embed=await self.__get_leaderboard(view.page), view=view)
```

Comment:
question: instead of changing response for everyone, should we instead allow `/leaderboard {page}`?

Reply:
Not necessary anymore